defender/download.js

```diff
@@ -162,7 +160,7 @@ function parseContractSentinel(item, notifications, autotasks) {
     abi,
     functionConditions,
     eventConditions,
-    txCondition: txConditions,
+    txCondition: txConditions[0].expression,
```

Comment:
Is it possible for `condition.txConditions` to be `undefined` or an empty Array?